### PR TITLE
[DOC] Filled in MultiMesh doc

### PR DESCRIPTION
**Pull request by clayjohn**

### CODE REVIEW

File: doc/classes/MultiMesh.xml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 		Provides high performance mesh instancing.
 	</brief_description>
 	<description>
-		MultiMesh provides low level mesh instancing. If the amount of [Mesh] instances needed goes from hundreds to thousands (and most need to be visible at close proximity) creating such a large amount of [MeshInstance] nodes may affect performance by using too much CPU or video memory.
-		For this case a MultiMesh becomes very useful, as it can draw thousands of instances with little API overhead.
+		MultiMesh provides low level mesh instancing. Drawing thousands of [MeshInstance] nodes can be slow becuase each object is submitted to the GPU to be drawn individually.
```
Comment (fire) [inline doc/classes/MultiMesh.xml]: becuase -> because

File: doc/classes/MultiMesh.xml [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+		 MultiMesh is much faster becuase it can draw thousands of instances with a single draw call, resulting in less API overhead.
```
Comment (fire) [inline doc/classes/MultiMesh.xml]: becuase -> because

File: doc/classes/MultiMesh.xml (hunk 1 above)
Comment (clayjohn) [reply doc/classes/MultiMesh.xml]: Thanks!

File: doc/classes/MultiMesh.xml [hunk 3]
```diff
@@ -4,8 +4,8 @@
 		Provides high performance mesh instancing.
 	</brief_description>
 	<description>
-		MultiMesh provides low level mesh instancing. If the amount of [Mesh] instances needed goes from hundreds to thousands (and most need to be visible at close proximity) creating such a large amount of [MeshInstance] nodes may affect performance by using too much CPU or video memory.
-		For this case a MultiMesh becomes very useful, as it can draw thousands of instances with little API overhead.
+		MultiMesh provides low level mesh instancing. Drawing thousands of [MeshInstance] nodes can be slow because each object is submitted to the GPU to be drawn individually.
+		 MultiMesh is much faster because it can draw thousands of instances with a single draw call, resulting in less API overhead.
```
Comment (akien-mga) [inline doc/classes/MultiMesh.xml]: Extra space at the start of the sentence.

File: doc/classes/MultiMesh.xml [hunk 4]
```diff
@@ -82,32 +84,45 @@
...
+			The [Color] passed into [set_instance_color] will use 4 floats. Use this for highest precision [Color].
 		</constant>
 		<constant name="CUSTOM_DATA_NONE" value="0" enum="CustomDataFormat">
+			Use when you are not using per-instance custom data.
 		</constant>
 		<constant name="CUSTOM_DATA_8BIT" value="1" enum="CustomDataFormat">
+			Compress custom_data into 8 bits when passing to shader. This uses less memory and can be faster, but loses precision.
 		</constant>
 		<constant name="CUSTOM_DATA_FLOAT" value="2" enum="CustomDataFormat">
+			The [Color] passed into [set_instance_custom_data] will use 4 floats. Use this for highest precision.
```
Comment (akien-mga) [inline doc/classes/MultiMesh.xml]: Method references are done with `[method <method_name>]`, so it should be `[method set_instance_custom_data]`.

File: doc/classes/MultiMesh.xml [hunk 5]
```diff
@@ -82,32 +84,45 @@
...
+			Use this when using 3D transforms.
 		</constant>
 		<constant name="COLOR_NONE" value="0" enum="ColorFormat">
+			Use when you are not using per-instance [Color]s.
 		</constant>
 		<constant name="COLOR_8BIT" value="1" enum="ColorFormat">
+			Compress [Color] data into 8 bits when passing to shader. This uses less memory and can be faster, but the [Color] loses precision.
 		</constant>
 		<constant name="COLOR_FLOAT" value="2" enum="ColorFormat">
+			The [Color] passed into [set_instance_color] will use 4 floats. Use this for highest precision [Color].
```
Comment (akien-mga) [inline doc/classes/MultiMesh.xml]: Same here.

**Comment by clayjohn** [comment]
Thanks @akien-mga!!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**